tests/shell-script-tests.sh

```
@@ -1476,6 +1476,151 @@ test_wt_cd_noninteractive_requires_name() {
+  select-window) touch "${TMUX_STUB_MARKER:?}" ;;
+  *)             exit 0 ;;
+esac
+STUB
+  chmod +x "$stub_dir/tmux"
+
+  output=$(
+    TMUX="$sandbox/fake-tmux-socket,1234,0" \
+    TMUX_STUB_PANE_PATH="$expected_path" \
+    TMUX_STUB_MARKER="$marker" \
+    HOME="$home_dir" \
+    PATH="$stub_dir:$FIXTURE_DIR/bin:$PATH" \
```

Shellcheck (0.11.0)

[info] PATH was modified in a subshell. That change might be lost.

(SC2031)